examples/common/load_dds.adb

```diff
@@ -0,0 +1,277 @@
+
+with System;
+with Interfaces;
+
+with Ada.Exceptions; use Ada.Exceptions;
+with Ada.IO_Exceptions; use Ada.IO_Exceptions;
+with Ada.Text_IO; use Ada.Text_IO;
+with Ada.Unchecked_Conversion;
+
+with GL.Types;
+with GL.Low_Level.Enums;
+with GL.Objects.Buffers;
+with GL.Objects.Textures.Targets;
+with GL.Objects.Textures.With_2D_Loader;
+with GL.Pixels;
+
+with IO;
+
+--  Extracted from SOIL.adb  Direct_Load_DDS
+procedure Load_DDS (File_Name  : String;
+                        theTexture : out GL.Objects.Textures.Texture) is
+    type t4_UBytes is array (1 .. 4) of GL.Types.UByte;
+    type t44_UBytes is array (1 .. 44) of GL.Types.UByte;
+    type tDDS_Data is array (GL.Types.UInt range <>) of GL.Types.UByte;
+
+    type tDDS_Header is record
+        Magic          : String (1 .. 4);
+        Height         : GL.Types.UInt;
+        Width          : GL.Types.UInt;
+        Linear_Size    : GL.Types.UInt;
+        Mip_Map_Count  : GL.Types.UInt;
+        Four_CC        : String (1 .. 4);
+        Byte_Size      : GL.Types.UInt := 128; --  bytes
+    end record;
```

Comment:
Why is this record named `tDDS_Header` but is loaded via `Load_DSS_Header`? DDS or DSS?